frontend/src/metabase/common/components/Pickers/CollectionPicker/hooks.ts

```diff
@@ -139,6 +141,33 @@ export const useRootCollectionPickerItems = (
       }
     }
 
+    if (tenantsEnabled && currentUser) {
+      collectionItems.push({
+        name: t`Shared Tenant Collections`,
+        id: "tenant",
+        namespace: "shared-tenant-collection",
+        here: ["collection", "card", "dashboard"],
+        description: null,
+        can_write: true,
```

Comment:
this should prevent saving in this collection, without preventing navigating into it

```suggestion
        can_write: false,
```